_includes/download.html

```diff
@@ -27,8 +27,8 @@
                         <h3>Workshops</h3>
                         <ul>
                           <li>Foam Roller Demos</li>
-                          <li>Yoga fundamentals</li>
-                          <li>Stretching</li>
+                          <li>Yoga Fundamentals</li>
+                          <li>Mindful Stretching</li>
```

Comment:
Sure

On Thursday, March 17, 2016, Jessie Keck <notifications@github.com> wrote:

> In _includes/download.html
> <https://github.com/bemobilewellness/bemobilewellness.github.io/pull/9#discussion_r56576219>
> :
>
> > @@ -27,8 +27,8 @@
> >                          <h3>Workshops</h3>
> >                          <ul>
> >                            <li>Foam Roller Demos</li>
> > -                          <li>Yoga fundamentals</li>
> > -                          <li>Stretching</li>
> > +                          <li>Yoga Fundamentals</li>
> > +                          <li>Mindful Stretching</li>
>
> This is not able to be merged in because Mindful Stretching is already in.
>
> [image: screen shot 2016-03-17 at 1 55 02 pm]
> <https://cloud.githubusercontent.com/assets/96776/13860913/e82b1c6e-ec47-11e5-891b-996ae861c053.png>
>
> I think the only change required here would be to just change fundamentals
> to Fundamentals.
>
> You may want to submit a new PR with that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bemobilewellness/bemobilewellness.github.io/pull/9/files/af4ca84b141fc9897f6abd41939ce6e3fe8e1f82#r56576219>
>


-- 
Vanessa Ragains
Be Mobile Wellness
(650) 516-7441
www.BeMobileWellness.com